electrum/lnutil.py

```diff
@@ -2022,3 +2023,28 @@ def _calculate_fee_msat(
         fee_msat = invoice_amount_msat * millionths_clamped // 1_000_000
         fee_msat = max(fee_msat, cutoff_clamped)
         return fee_msat
+
+    @classmethod
+    def reverse_from_total_amount(cls, total_amount_msat: int, config: 'SimpleConfig') -> int:
```

Comment:
nit: at least `config` should be a kwarg.
In my view, every arg to a function - that is not obvious just from reading the name of the function - should be a kwarg.